src/parser.js

```diff
@@ -25,7 +25,10 @@ class queryParser{
 		
 		// Are we currently seeking new delimiter
 		this.seekingDelimiter = false;
-		
+
+		// Does the sql set change delimiter?
+                this.hasDelimiter = queriesString.includes('DELIMITER ');
```

Comment:
Please use tabs instead of spaces to keep the file congruent. Also, the "delimiter" keyword is case insensitive. Please account for lower case as well.. `queriesString.toLowerCase().includes('delimiter ');`